sdk/nodejs/package.json

```diff
@@ -13,8 +13,13 @@
     "peerDependencies": {
         "@openfeature/server-sdk": "^1.11.0"
     },
+    "peerDependenciesMeta": {
+        "@openfeature/server-sdk": {
```

Comment:
i thought you didnt want to make this optional?

Reply:
yea I keep flipflopping on it, I think I'll keep it optional to avoid the yarn warnings.

Reply:
that does mean that you will always have to install those packages yourself when you want to use openfeature

Reply:
Yes, the trade-off is either: 
- make them optional, and node / pnpm / yarn users would need to install the OF deps to use OF.
- make them non-optional, node / pnpm users don't need to do anything, yarn users will get a warning if they don't have them installed.

When I put it that way, option 1 sounds better.

Reply:
it seems weird that someone using openfeature wouldn't already be intending to install those packages directly?